packs/data/feats.db/blade-of-justice.json

```diff
@@ -30,18 +29,12 @@
             ]
         },
         "rules": [
-            {
-                "domain": "damage-roll",
-                "key": "RollOption",
-                "option": "blade-justice",
-                "toggleable": true
-            },
             {
                 "diceNumber": 2,
                 "key": "DamageDice",
                 "predicate": {
                     "all": [
-                        "blade-justice"
+                        "target:trait:evil"
```

Comment:
This doesn't apply to all strikes vs evil, the feat is a 2 action ability so we have to keep it as a toggle.

Reply:
Oops! Thanks for catching that one